styles.css

```diff
@@ -0,0 +1,330 @@
+    font-size: 15px;
+    font-family: 'Urbanist';
+    line-height: 23px;
+}
+.arrow{
+    display: flex;
+    justify-content: flex-end;
+    gap: 10px;
+    cursor: pointer;
+    color: var(--teal);
+    font-size: 16px;
+    font-weight: 600;
+    align-items: center;
+    padding-right: 10px;
+}
+.arrow p{
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 23px;
+    font-family: 'Gilda Display';
+    color: var(--teal);
+}
+.guide-card:hover{
+    background: var(--teal);
+    color: var(--white);
+}
+.guide-card:hover .arrow {
+    color: var(--white);
+    background: var(--teal);
+}
+.guide-card:hover .arrow p{
+    color: var(--white);
+    background: var(--teal);
+}
+.map{
+    padding: 20px;
+    text-align: left;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+.map h2{
+    font-family: 'Gilda Display';
+    font-weight: 400;
+    font-size: 40px;
+}
+.map h2 span{
+    color: var(--teal);
+}
+.map p{
+    font-family: 'Urbanist';
+    font-weight: 500;
+    font-size: 16px;
+}
+.map p span{
+    color: var(--teal);
+}
+.map img{
+}
+.video h2{
+    font-family: 'Gilda Display';
+    font-weight: 400;
+    font-size: 40px;
+    text-align: center;
+}
+.video img{
+    width: 100%;
+    text-align: center;
+}
+.footer-top{
+    border-bottom: 1px solid var(--light-grey);
+    padding-bottom: 40px;
+    padding-left: 20px;
+}
+.footer-top h2{
+    font-weight: 400;
+    font-size: 40px;
+    font-family: 'Gilda Display';
+    text-align: left;
+}
+button{
+    color: var(--white);
+    background-color: var(--teal);
+    font-family: 'Urbanist';
+    font-size: 18px;
+    font-weight: 600;
+    padding: 16px 32px;
+    gap: 10px;
+    border: 0;
+    justify-content: left;
+}
+
+.footer-bottom h4{
+    font-family: 'Gilda Display';
+    font-size: 16px;
+    font-weight: 400;
+    letter-spacing: 0.1em;
+    color: var(--teal);
+    text-align: left;
+    padding-top: 20px;
+    padding-left: 20px;
+    text-align: left;
+
+}
+.footer-bottom p{
+    font-family: 'Urbanist';
+    text-align: left;
+    font-size: 16px;
+    font-weight: 500;
+    text-align: left;
+    padding-bottom: 20px;
+    padding-left: 20px;
+}
+.footer-icons{
+    text-align: center;
+}
+.navs li{
+    text-align: center;
+    display: inline-flex;
+    font-size: 16px;
+    font-weight: 600;
+    font-family: 'Urbanist';
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-bottom: 30px;
+}
+/* fonts */
+@font-face{
+    font-family: 'Urbanist';
+    src: url(fonts/Urbanist-Regular.ttf);
+}
+@font-face {
+    font-family: 'Gilda Display';
+    src: url(fonts/GildaDisplay-Regular.ttf);
+}
+/* mobile styles */
+@media (max-width: 650px){
+    body{
+        background-color: #fff;
+        box-shadow: rgba(255, 255, 255, 1);
+        margin: 0px;
+        padding: 0px;
+        border: 0px;
+    }
+
+
+/* tablet */
+
+
+
+/* desktops */
+.nav ul{
+    list-style-type: none;
+    margin: 0px;
+}
```

Comment:
Hi @EscHaleem,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    <img src="https://camo.githubusercontent.com/7cf1ba8db9cd08f454a9651e48128c064a4440a59881658c8593a41a8de243f3/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966" alt="Remote Collaboration" style="width:40%;">
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
-  **Good use of HTML Semantic** 👍
-  **Professional PR title** 👍
-  **Professional PR description** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️

#  Adjust the width of mirror lake card

- The Mirror Lake should have the same width with the remaining cards
![Step 1 screenshot](https://images.tango.us/workflows/63b80b91-68d1-4812-a390-d969f444ecc9/steps/e69bf890-c0fa-4da5-b590-93c8a17d62d9/38054833-6abe-425a-960d-29d866047611.png?crop=focalpoint&fit=crop&fp-x=0.3054&fp-y=0.3898&fp-z=1.0063&w=1200&border=2%2CF4F2F7&border-radius=8%2C8%2C8%2C8&border-radius-inner=8%2C8%2C8%2C8&blend-align=bottom&blend-mode=normal&blend-x=0&blend-w=1200&blend64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmstdjIucG5n&mark-x=7&mark-y=7&m64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL2JsYW5rLnBuZz9tYXNrPWNvcm5lcnMmYm9yZGVyPTYlMkNGRjc0NDImdz03MjMmaD04OTQmZml0PWNyb3AmY29ybmVyLXJhZGl1cz0xMA%3D%3D)

<br/>

#  Remove the hover effect on all the cards

- Remove the hover effect on each card

![Step 1 screenshot](https://images.tango.us/workflows/ea62b2b5-c97d-4612-beee-40037d0c1c23/steps/84f41706-70a9-4c4f-9c28-c5b8fc6697c9/90acd5a6-8fde-4798-91a9-9d4ab8ec8916.png?crop=focalpoint&fit=crop&fp-x=0.5000&fp-y=0.5000&w=1200&border=2%2CF4F2F7&border-radius=8%2C8%2C8%2C8&border-radius-inner=8%2C8%2C8%2C8&blend-align=bottom&blend-mode=normal&blend-x=0&blend-w=1200&blend64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmstdjIucG5n&mark-x=5&mark-y=16&m64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL2JsYW5rLnBuZz9tYXNrPWNvcm5lcnMmYm9yZGVyPTQlMkNGRjc0NDImdz0zNjYmaD0zMjcmZml0PWNyb3AmY29ybmVyLXJhZGl1cz0xMA%3D%3D)

- This is how your cards should be display

![Step 2 screenshot](https://images.tango.us/workflows/ea62b2b5-c97d-4612-beee-40037d0c1c23/steps/9e31b014-b506-4101-95bb-c481993139ba/0995e656-e21d-4dd6-aa2b-a88e7320c7f9.png?crop=focalpoint&fit=crop&fp-x=0.5000&fp-y=0.5000&w=1200&border=2%2CF4F2F7&border-radius=8%2C8%2C8%2C8&border-radius-inner=8%2C8%2C8%2C8&blend-align=bottom&blend-mode=normal&blend-x=0&blend-w=1200&blend64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmstdjIucG5n)

<br/>

#  Make sure your ul li are display like the figma design

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._